includes/class-wcs-cart-renewal.php

```diff
@@ -391,7 +391,7 @@ public function cart_items_loaded_from_session( $cart ) {
 				continue;
 			}
 
-			if ( isset( $item[ $this->cart_item_key ]['renewal_order_id'] ) && ! 'shop_order' == get_post_type( $item[ $this->cart_item_key ]['renewal_order_id'] ) ) {
+			if ( isset( $item[ $this->cart_item_key ]['renewal_order_id'] ) && ! 'shop_order' === WC_Data_Store::load( 'order' )->get_order_type( $item[ $this->cart_item_key ]['renewal_order_id'] ) ) {
```

Comment:
```suggestion
			if ( isset( $item[ $this->cart_item_key ]['renewal_order_id'] ) && 'shop_order' !== WC_Data_Store::load( 'order' )->get_order_type( $item[ $this->cart_item_key ]['renewal_order_id'] ) ) {
```

Reply:
@brucealdridge I believe the original code intends to remove cart item if order type is not `shop_order`. The `!` has been misplaced there before this PR's change. Could you make sure I got that right?

Reply:
Good spotting @shendy-a8c I've done some testing and confirmed that if we commit your fix manual renewals will break. 
I've [created an issue](https://github.com/Automattic/woocommerce-subscriptions-core/issues/170) to follow up on later.

Reply:
Thanks for catching that!